configs/components/cpp-pcp-client.rb

```diff
@@ -46,7 +46,10 @@
     pkg.build_requires "pl-cmake"
     pkg.build_requires "pl-boost"
 
-    if platform.is_cisco_wrlinux?
+    # SLES and EL-4 have old versions of gettext on them; until we build our own,
```

Comment:
Also might be worth noting what version of gettext is required here.

Sles 10 has gettext 0.14.5-13.2
Sles 11 SP3 has gettext 0.17-61.40
Sles 12 has gettext 0.19.2-1.103